### PR TITLE
Fix theme category listing issues (fix #2025)

### LINKED ISSUE #2025: Themes listings have layout issues
Steps to reproduce:
1. Load any theme category page on AMO-dev  (i.e. https://addons-dev.allizom.org/en-US/firefox/themes/film-and-tv)
2. Observe the "Up & Coming" section

Expected results:
There are no layout issues.

Actual results:
There is a white padding after the columns.

Notes/Issues:
**I think same thing is happening in theme details page for "more.." section.**

Verified on AMO-dev FF45(Win 7). 
Screenshot for this issue:

### PR DESCRIPTION
**Pull request by tofumatt**
### After

screenshot 2016-03-30 12 17 57
screenshot 2016-03-30 12 17 55

### CODE REVIEW

**Comment by tofumatt** [comment]
Note: This fixed a link colour outlier and the alignment of the "View All $THEME_CATEGORY" link too.

**Comment by muffinresearch** [comment]
Needs rebasing.

**Comment by tofumatt** [comment]
Yeah, I messed up a closing bracket too. On it!

**Comment by tofumatt** [comment]
Rebased and ready for r?

**Comment by muffinresearch** [comment]
Looks like you've got conflict markers in the diff

**Comment by tofumatt** [comment]
Oops, I clearly need lunch! ;-)

**Comment by tofumatt** [comment]
Alright, should be fixed!

File: static/css/restyle.less [hunk 1]
```diff
@@ -96,8 +96,9 @@ h2 {
 h1,
 .primary h2,
 hgroup h2.addon,
-.personas-featured h3 {
-  color: @default-font-color;
+.personas-featured h3,
+#featured-addons .featured-inner h3 {
+  color: @header-font-color;
```
Comment (muffinresearch) [inline static/css/restyle.less]: These headeings are supposed to be #333 - which is getting removed with this change by the looks of it.

Comment (tofumatt) [inline static/css/restyle.less]: Oh shoot, you're right. I'll fix that.

**Comment by muffinresearch** [comment]
r+wc

**Pull request merged**